gemrb/GUIScripts/bg2/Start.py

```diff
@@ -21,32 +21,32 @@
 import GameCheck
 from GUIDefines import SV_SAVEPATH
 
-skip_videos = False
+skipVideos = False
 
 def RunStart2(isTOB):
-	global skip_videos
-	
+	global skipVideos
+
+	skipIntro = GemRB.GetVar ("SeenIntroVideos")
```

Comment:
Won't this be None on first run, breaking the anding lower?

Reply:
No, the `OnLoad` sets this to 1 before getting here, and it's not executed when all intro videos are skipped.